packages/codemods/src/transforms/popper.ts

```diff
@@ -0,0 +1,21 @@
+import { API, FileInfo } from 'jscodeshift';
+import { getImportInfo, renameProp } from '../codemod-helpers';
+import { JSCodeShiftOptions } from '../types';
+
+export const parser = 'tsx';
+
+const RENAME_MAP = {
+  offsetDistance: 'offsetByMainAxis',
+  offsetSkidding: 'offsetByCrossAxis',
+};
+
+export default function transformer(file: FileInfo, api: API, options: JSCodeShiftOptions) {
+  const { alias } = options;
+  const j = api.jscodeshift;
+  const source = j(file.source);
+  const { localName } = getImportInfo(j, file, 'Popper', alias);
```

Comment:
```suggestion
  const { localName } = getImportInfo(j, file, 'unstable_Popper', alias);
```

в `getImportInfo` не увидил, чтобы учитывалось, что как `unstable` экспортируется